src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -2808,4 +2808,22 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Convert_to_record" xml:space="preserve">
     <value>Convert to record</value>
   </data>
+  <data name="Introduce_parameter" xml:space="preserve">
+    <value>Introduce parameter</value>
+  </data>
+  <data name="Introduce_parameter_for_0" xml:space="preserve">
+    <value>Introduce parameter for '{0}'</value>
+  </data>
+  <data name="Introduce_parameter_for_all_occurrences_of_0" xml:space="preserve">
+    <value>Introduce parameter for all occurrences of '{0}'</value>
+  </data>
+  <data name="into_new_overload_of_0" xml:space="preserve">
+    <value>into new overload of '{0}'</value>
+  </data>
+  <data name="into_extracted_method_to_invoke_at_callsites" xml:space="preserve">
+    <value>into extracted method to invoke at callsites</value>
+  </data>
+  <data name="and_update_callsites_directly" xml:space="preserve">
+    <value>and update callsites directly</value>
```

Comment:
"call sites" (applies to this loc string, and the one above).

Reply:
changed